TMGToolbox/src/XTMF_internal/multi_class_road_assignment.py

```diff
@@ -33,17 +34,25 @@
 
     V 1.1.1 Updated to allow for multi-threaded matrix calcs in 4.2.1+
         
-'''
+"""
+
 
 import inro.modeller as _m
 import traceback as _traceback
 import multiprocessing
 from contextlib import contextmanager
-from contextlib import nested
+import six
+if six.PY3:
+    _m.InstanceType = object
+    _m.TupleType = object
+    _m.ListType = object
```

Comment:
Even if we are not using the list type call
```Python
 _util.initalizeModellerTypes(_m).
```

Reply:
fixed